Makefile

```diff
@@ -131,7 +131,15 @@ verify-genproto:
 
 .PHONY: verify-yamllint
 verify-yamllint:
+ifeq (, $(shell which yamllint))
+	@echo "Installing yamllint..."
+	python3 -m venv bin/python
```

Comment:
Solution using local virtualenv is nice and thought out. Not a great fun of adding python dependency, still we don't have a choice as we decided to use a python tool for linting.

Would be interested if there is any other tool that has both fix (yamlfmt) and (yamllint) functionality. Maybe we should just use yamlfmt and implement our own validation based on diff like we did for gen proto. Don't want to block migration to test-infra on this, so we can use python for now.

Reply:
Agreed - We should move away from `yamllint` in future to avoid bringing in python dependencies.